MapboxCoreNavigation/Constants.swift

```diff
@@ -62,12 +62,26 @@ public var MaxSecondsSpentTravelingAwayFromStartOfRoute: TimeInterval = 3
 
 
 /**
- Distance in meters for the minimum length of a step for giving a `medium` alert.
+ Distance in meters for the minimum length of a step for giving a `medium` alert while using the `MBDirectionsProfileIdentifier` `.automobileAvoidingTraffic` or `.automobile`
  */
 public var RouteControllerMinimumDistanceForMediumAlertDriving: CLLocationDistance = 400
+
+
+/**
+ Distance in meters for the minimum length of a step for giving a `medium` alert while using the `MBDirectionsProfileIdentifier` `.cycling`
```

Comment:
Add a period to the end of every documentation comment. This makes things more consistent in the event that we need to document any symbol with multiple sentences.